packages/template-retail-react-app/app/pages/login/index.jsx

```diff
@@ -48,34 +56,54 @@ const Login = () => {
         {enabled: !!customerId && !isServer, keepPreviousData: true}
     )
     const mergeBasket = useShopperBasketsMutation('mergeBasket')
+    const [currentView, setCurrentView] = useState(initialView)
```

Comment:
Similar to the logic in [use-auth-modal.js](https://github.com/SalesforceCommerceCloud/pwa-kit/blob/feature-passwordless-social-login/packages/template-retail-react-app/app/hooks/use-auth-modal.js), I am using `currentView` to switch from the login page to the email confirmation page. This allows us to use the same `form` object in both the `LoginForm` and the `PasswordlessEmailConfirmation` components.